src/group-configurations/group-configuration-sidebar/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
[nit]: Do we need this unnecessary React import?

Reply:
Yes, there is  <React.Fragment key={title}>